src/test/e2e/Helpers/Selectors.ts

```diff
@@ -17,6 +17,15 @@ module.exports = {
   SearchOptionsTitle: 'h1.govuk-fieldset__heading',
   RadioButton: 'div.govuk-radios__item',
 
+  // Search page selectors
```

Comment:
these look like they are not used anywhere, can we remove this please?

Reply:
removed